src/payment/data/utils.test.js

```diff
@@ -370,3 +372,50 @@ describe('chainReducers([reducers])', () => {
     expect(firstStateResult.second_reducer_value).toEqual(undefined);
   });
 });
+
+describe('createCustomRoutine', () => {
```

Comment:
Tests that our custom routines do everything we expect them to do and provides all the expected `_STAGES` as well as the right `_PREFIX`. (the base name of the routine)